airflow/cli/commands/task_command.py

```diff
@@ -213,6 +213,29 @@ def task_list(args, dag=None):
         print("\n".join(sorted(tasks)))
 
 
+@cli_utils.action_logging
+def task_states_for_dag_run(args):
+    """Get the status of all task instances in a dag run"""
+    if not args.dag_id or not args.execution_date:
+        raise AirflowException("dag_id and execution_date are mandatory.")
+    session = settings.Session()
+    tis = session.query(
+        TaskInstance.dag_id,
+        TaskInstance.execution_date,
+        TaskInstance.task_id,
+        TaskInstance.state,
+        TaskInstance.start_date,
+        TaskInstance.end_date).filter(
+        TaskInstance.dag_id == args.dag_id,
+        TaskInstance.execution_date == args.execution_date).all()
+    session.close()
+    if len(tis) == 0:
+        raise AirflowException("dag run does not exist.")
+    for ti in tis:
+        print("dag={0}, exec_date={1}, task={2}, state={3}, start_date={4}, end_date={5}".format(
```

Comment:
Can you use tabulate?  All CLI lists should use it to enable data display in various formats, e.g. CSV.
Referencec: 
https://github.com/apache/airflow/blob/master/airflow/cli/commands/pool_command.py#L31-L33

Reply:
fixed